contracts/test/v0.8/foundry/vrf/VRFV2PlusWrapper.t.sol

```diff
@@ -135,13 +153,46 @@ contract VRFV2PlusWrapperTest is BaseTest {
   // VRFV2PlusWrapperConsumerBase events
   event LinkTokenSet(address link);
 
+  // SubscriptionAPI events
+  event SubscriptionConsumerAdded(uint256 indexed subId, address consumer);
+
   function testVRFV2PlusWrapper_ZeroAddress() public {
     vm.expectRevert(VRFConsumerBaseV2Plus.ZeroAddress.selector);
-    new VRFV2PlusWrapper(address(0), address(0), address(0));
+    new VRFV2PlusWrapper(address(s_linkToken), address(0), address(0), uint256(0));
   }
 
-  function testSetLinkNativeFeed() public {
-    VRFV2PlusWrapper wrapper = new VRFV2PlusWrapper(address(s_linkToken), address(0), address(s_testCoordinator));
+  function testVRFV2PlusWrapperWithExistingSubscriptionId() public {
```

Comment:
Arent all tests testing wrapper with existing subId now? Is this test doing anything different

Reply:
Will rename it to `testCreationOfANewVRFV2PlusWrapper` to be ever more explicit.